RedHat_NLP/Deliverable 1.ipynb

```diff
@@ -0,0 +1,285 @@
gPR39zDzro6I9ohob2lpqVTJZmZ1oeKhIOlISbMGngPvA54A1gOXpskuBdZVujYzs3pXje6j+cA3JQ28/1ci4ruSfgLcJeky4HngoirUZmZW1yoeChHxLHBiifa9wOmVrsfMzF4zmQ5JNTOzKnMomJlZxqFgZmYZh4KZmWUcCmZmlnEomJlZxqFgZmYZh4KZmWUcCmOw90A3G7ftZ++B7mqXYmaWi2pdJXXKWde5g49/fSONaqAv+vn8BSdyTtvCapdlZjahvKVQhr0Hurnirk66e4NXevro7g3+/K5ObzGYWc1xKJRh0y9eord/cFtvf6HdzKyWOBTKUvJ+PyO0m5lNTQ6FMpxwzFE0NWpQW1OjOOGYo6pUkZlZPhwKZZg7s5nrLjyR5mkN/MYRjTRPa+C6C09k7szmapdmZjahfPRRmc5pW8iKJfPYvu8grXNmOBDMrCY5FEax90B3FgQD9v36VYeDmdWkugyFmx94hm917eTc5Qv48HuWDjvdus4dXLW2i6aGBg719hERNEp09wXTmwo9b9eev9xbEGZWMxQxdY+gaW9vj46OjjHN89ufuoeDva8t84xp4sm/Pfuw6fYe6GbF5+7nUE//YeOKNTUKAdMaG+jrDz5/wXKf1GZmk5qkRyOivdS4utrRfPMDzwwKBICDvcHNDzxz2LTb9x2kqWH01dPTF7zaF7zyah/dvf1c8X83+qQ2M5uy6ioUvtW1s+z21jkz6OkfeSuhlJ6+YNMvXhzzfGZmk8GkCwVJZ0p6StIWSVdP5Gufu3xB2e1zZzZz7fnLmd7UwKzmaTQ1imkN0JzOV5je1MARjcOtPg3TbmY2uU2qHc2SGoEvAu8FtgM/kbQ+IjZPxOtf8LZj+ft7ny7ZXsrQw1Ch0K105BGN/PrVPo48opEz//eDgy6BMa0BTjjmDRNRrplZxU2qUABOAbZExLMAku4EVgITEgrb9x1kVvM0Xu7uzdpmNU9j+76Dwx41NHdm86BxQ6e7/qI2rhxy9VQfgWRmU9VkC4WFwLai4e3A7xVPIGkVsArg2GNL/8IfTqn9BD39/YPOQRgrn9RmZrVk0u1TGE1ErI6I9ohob2lpGdO8Q/cTTG9q4Nrzl7/uL/K5M5s5cdFsB4KZTXmTbUthB7CoaLg1tU0Y/7I3MxveZAuFnwBLJR1HIQwuBv5wot9k6H4CMzMrmFShEBG9kj4C3As0ArdFxKYql2VmVjcmVSgARMQ9wD3VrsPMrB5NuR3NZmaWH4eCmZllHApmZpaZ0pfOlrQHeH4cs84DfjnB5UxFXg9eBwO8HuprHbwpIkqe6DWlQ2G8JHUMdy3xeuL14HUwwOvB62CAu4/MzCzjUDAzs0y9hsLqahcwSXg9eB0M8HrwOgDqdJ+CmZmVVq9bCmZmVkLdhUKet/uczCRtlfS4pE5JHantaEn3SXom/Z1T7TonmqTbJO2W9ERRW8nlVsGN6bPRJenk6lU+sYZZD5+RtCN9JjolnV007hNpPTwl6YzqVD2xJC2S9ICkzZI2Sfpoaq+7z8NI6ioUim73eRawDPigpGXVraqi3hMRbUWH3V0NbIiIpcCGNFxrbgfOHNI23HKfBSxNj1XATRWqsRJu5/D1AHBD+ky0peuOkf4nLgZOSPP8U/rfmep6gSsiYhlwKnB5WtZ6/DwMq65CgaLbfUbEq8DA7T7r1UpgTXq+Bji3irXkIiIeBH41pHm45V4J3BEFDwGzJS2oTKX5GmY9DGclcGdEdEfEc8AWCv87U1pE7IyIx9Lzl4EnKdztse4+DyOpt1AodbvPhVWqpdIC+J6kR9MtTQHmR8TO9PwFYH51Squ44Za7Hj8fH0ldI7cVdR/W/HqQtBg4CXgYfx4GqbdQqGfviIiTKWwSXy7pncUjo3AYWt0dilavy53cBBwPtAE7geuqW05lSJoJrAU+FhEvFY+r888DUH+hkPvtPieriNiR/u4GvkmhO2DXwOZw+ru7ehVW1HDLXVefj4jYFRF9EdEP3MJrXUQ1ux4kNVEIhC9HxDdSsz8PReotFLLbfUo6gsLOtPVVril3ko6UNGvgOfA+4AkKy35pmuxSYF11Kqy44ZZ7PXBJOurkVODFom6FmjOkf/w8Cp8JKKyHiyU1p1vjLgUeqXR9E02SgFuBJyPi+qJR/jwUi4i6egBnA08DPwM+We16KrTMbwY2psemgeUG5lI42uIZ4PvA0dWuNYdl/yqFrpEeCn3Clw233IAoHJ32M+BxoL3a9ee8Hr6UlrOLwhfggqLpP5nWw1PAWdWuf4LWwTsodA11AZ3pcXY9fh5GeviMZjMzy9Rb95GZmY3AoWBmZhmHgpmZZRwKZmaWcSiYmVnGoWBWRFJf0VVDO9PlEIab9t/T38XFVx81m8qmVbsAs0nmYES0lTNhRPzH8b6JpMaI6Bvv/GZ58ZaC2QgkzZS0QdJj6X4UK4vGHSgx/X+R9I9Fw3dLevfA9JKuk7QReLukP5b0SNoiuVlSY3rcLumJ9H7/oxLLaTbAWwpmg82Q1JmePwdcCJwXES9Jmgc8JGl9jO+szyOBhyPiCkm/DVwFrIiIHkn/BPwRhTPOF0bE7wBImv26l8hsDBwKZoMN6j5KF1D7u3RV2X4Kl06eT+ESy2PVR+FibACnA78L/KRwSR5mULgQ278Ab5b0D8C3ge+NcznMxsWhYDayPwJagN9Nv+i3AtNHmL6Xwd2yxdMeKtqPIGBNRHxi6AtIOhE4A/hT4CLgv46/fLOx8T4Fs5EdBexOgfAe4E2jTL8VaJPUIGkRw9+xbANwgaQ3Qnaf4DelLqqGiFgLfAqoi/sC2+ThLQWzkX0Z+BdJjwMdwE9Hmf5HFPZFbKZwu8fHSk0UEZslfYrC3fAaKFy99HLgIPDPqQ3gsC0Jszz5KqlmZpZx95GZmWUcCmZmlnEomJlZxqFgZmYZh4KZmWUcCmZmlnEomJlZxqFgZmaZ/w80b9KczfD0mAAAAABJRU5ErkJggg==\n",
```

Comment:
You should add a markdown cell below this graph explaining what it tells us about our data.